test/test_helper.rb

```diff
@@ -11,4 +11,11 @@
 
 require_relative 'helpers/rubocop_helper'
 
+require 'statsd/instrument/strict' if ENV['STATSD_STRICT_MODE']
+
+def strict_mode_enabled?
```

Comment:
Not sure i'm fan of having this defined as a main method. Can't it be inside some test helper?

Reply:
Fixed